website/docs/r/policy_assignment.html.markdown

```diff
@@ -93,6 +97,10 @@ The following attributes are exported:
 
 * `id` - The Policy Assignment id.
 
+* `identity_principal_id` - The principal ID of the resource identity. Only relevant if using `SystemAssigned` as `identity_type`.
+
+* `identity_tenant_id` - The tenant ID of the resource identity. Only relevant if using `SystemAssigned` as `identity_type`.
```

Comment:
(as above) since this'll be in a block could we make this:

```
---

An `identity` block exports the following:

* `principal_id` - The Principal ID of this Policy Assignment.

* `tenant_id` - The Tenant ID of this Policy Assignment.
```

Reply:
👍